src/amo/components/AddonHead/index.js

```diff
@@ -133,7 +141,8 @@ export class AddonHeadBase extends React.Component<InternalProps> {
   }
 
   render() {
-    const { addon } = this.props;
+    const { _getAddonJsonLinkedData, addon, currentVersion } = this.props;
+    invariant(_getAddonJsonLinkedData, '_getAddonJsonLinkedData is required.');
```

Comment:
Do you need this invariant statement because of flow?

Reply:
Yes, because it's optional Flow thinks it could be `undefined` here. I guess Flow doesn't understand `defaultProps`. I can remove the `invariant` if I create a new const here and set it to `_getAddonJsonLinkedData || getAddonJsonLinkedData` so I've done that in the next commit, but I feel like that's a bit ugly.

Reply:
hum, interesting. What if you move `_getAddonJsonLinkedData` into the `InternalProps` and make it not optional?

Reply:
That works, thanks!